### ISSUE TITLE
List other Kubo RPC API clients

### ISSUE THREAD

**Issue by BigLep** [ipfs/ipfs-docs#1194]
After https://github.com/ipfs/ipfs-docs/pull/1185 , we should update the equivalent of https://docs.ipfs.io/reference/http/api/ to highlight that there are other implementations.

| Language      | Client library                                            | Status              |
|---------------|-----------------------------------------------------------|---------------------|
| Go            | https://github.com/ipfs/go-ipfs-api                       | Active              |
| Java          | https://github.com/ipfs-shipyard/java-ipfs-http-client    | Active              |
| JavaScript    | https://github.com/ipfs/js-ipfs/tree/master/packages/ipfs-http-client | Active |
| Python        | https://github.com/ipfs-shipyard/py-ipfs-http-client      | Active |
| Scala         | https://github.com/ipfs-shipyard/scala-ipfs-api           | Inactive |
| Clojure       | https://github.com/keorn/clj-ipfs-http-client             | Active |
| Clojurescript | https://github.com/district0x/cljs-ipfs-http-client | Active |
| Haskell       | https://github.com/davidar/hs-ipfs-api                    | Inactive |
| Swift         | https://github.com/ipfs-shipyard/swift-ipfs-http-client   | Active |
| CommonLisp    | https://github.com/WeMeetAgain/cl-ipfs-api                | Inactive |
| Rust          | https://github.com/ferristseng/rust-ipfs-api              | Active |
|               | https://github.com/gkbrk/rust-ipfs-api                    | Inactive |
|               | https://github.com/rmnoff/rust-ipfs-api                   | Inactive |
|               | https://github.com/rschulman/rust-ipfs-api                | Inactive |
| Ruby          | https://github.com/Fryie/ipfs-ruby                        | Inactive |
|               | https://github.com/tbenett/ruby-ipfs-http-client          | Active   |
| Mac Automator | https://github.com/NeoTeo/ipfs-osx-service                | Inactive |
| Pharo         | https://github.com/khinsen/ipfs-pharo                     | Active   |
| PHP           | https://github.com/cloutier/php-ipfs-api                  | Inactive |
|               | https://github.com/digitalkaoz/php-ipfs-api               | Inactive |
| C#            | https://github.com/jeremy-ellis-tech/net-ipfs-http-client | Inactive |
|               | https://github.com/richardschneider/net-ipfs-http-client  | Active   |
| C++           | https://github.com/vasild/cpp-ipfs-api                    | Active   |
| Erlang        | https://github.com/hendry19901990/erlang-ipfs-http-client | Inactive |

This list was grabbed from https://github.com/ipfs/ipfs/tree/cleanup/root-readme#http-client-libraries

**Comment by lidel** [comment]
Adding a note that we want to tackle this AFTER we clean up and rename GO and JS libraries, tracked in:
-  `go-kubo-rpc-client` (https://github.com/ipfs/kubo/issues/9124)
-  `js-kubo-rpc-client` (https://github.com/ipfs/kubo/issues/9125)

**Comment by siiky** [comment]
I had this from a while back: https://github.com/ipfs/ipfs/pull/480

Should I open a PR here instead?

**Comment by BigLep** [comment]
@ElPaisano : it's true that https://github.com/ipfs/kubo/issues/9142 isn't completed.  This will likely take a month or so.

@lidel : that said, I wonder if we should get this docs page started so folks like @siiky and @ElPaisano can contribute any Kubo RPC clients they have.  Do you think we need to hold off ont his until https://github.com/ipfs/kubo/issues/9142 is completed?

**Comment by BigLep** [comment]
@lidel : do you think it makes sense for this content to be in https://docs.ipfs.tech/reference/kubo/rpc/ or in the Kubo repo itself?  (I believe ideally Kubo would have its own docs site with this kind of info rather than being in docs.ipfs.tech, but I know we're also trying to be realistic to where we are.)

**Comment by ElPaisano** [comment]
Triaging issues here @BigLep @lidel:

Is this still relevant? Did the team end up adding this in the Kubo docs? Depriortizing and setting to `status\deferred` until I hear back. Thank you

**Comment by BigLep** [comment]
@ElPaisano : This is still relevant.  I think we can get started and then do further improvements/followups once https://github.com/ipfs/kubo/issues/9124 lands.

**Comment by ElPaisano** [comment]
Confirming with the team that this is still blocked, see https://github.com/ipfs/kubo/issues/9124#issuecomment-1687222254

**Comment by ElPaisano** [comment]
Team confirmed that work can start, see https://github.com/ipfs/kubo/issues/9124
